src/views/profile/followingList.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Moment from "react-moment";
+import ReactTimeAgo from "react-time-ago";
```

Comment:
What is this and how is it used??

Reply:
Hey @wasibani-roy, the react time ago library is used for generating localized relative time strings. When the react-time-ago package is installed the javascript-time-ago package is installed automatically along with it. This works well together with the moment package to achieve the time since feature on dates.